setup.cfg

```diff
@@ -1,5 +1,5 @@
 [metadata]
-version = 5.1.2rc1.dev0
+version = 5.1.2rc2
```

Comment:
```suggestion
version = 5.1.2rc1.dev0
```

Reply:
That's confusing to be honest ... but I did